$OpenBSD$

OpenBSD paths: /etc/opt/simplex-xftp -> /etc/simplex/xftp-server,
/var/opt/simplex-xftp -> /var/simplex/xftp-server.

--- a/apps/xftp-server/Main.hs
+++ b/apps/xftp-server/Main.hs
@@ -7,10 +7,10 @@
 import XFTPWeb (xftpGenerateSite)
 
 defaultCfgPath :: FilePath
-defaultCfgPath = "/etc/opt/simplex-xftp"
+defaultCfgPath = "/etc/simplex/xftp-server"
 
 defaultLogPath :: FilePath
-defaultLogPath = "/var/opt/simplex-xftp"
+defaultLogPath = "/var/simplex/xftp-server"
 
 logCfg :: LogConfig
 logCfg = LogConfig {lc_file = Nothing, lc_stderr = True}
